### ISSUE TITLE
Bug: Changing more than one bitmapText per animation tick updates only one of the texts in v8-alpha-2

### ISSUE THREAD

**Issue by GDX64** [pixijs/pixijs#9619]
### Current Behavior

Changing more than one bitmapText per animation tick updates only one of the texts

### Expected Behavior

It should update on the screen all the texts and not just one

### Steps to Reproduce

``` typescript
const c = new Container();
  const textArr = [...new Array(10)].map(() => {
    const text = new Text({
      text: 'hello',
      renderMode: 'bitmap',
      style: { fontFamily: 'sans-serif', fontSize: 13 },
    });
    text.tint = 0x000000;
    text.x = Math.random() * 700;
    text.y = Math.random() * 600;
    c.addChild(text);
    return text;
  });
  const app = new Application();
  await app.init({
    element: canvas.value as any,
    backgroundColor: 0xffffff,
    resizeTo: canvas.value,
  });
  app.stage.addChild(c);
  let i = 0;
  app.ticker.add(() => {
    textArr.forEach((text) => {
      text.text = Math.random().toFixed(6);
    });
  });
```

### Environment

- **`pixi.js` version**: *v8-alpha-2*
- **Browser & Version**: *Chrome 115*
- **OS & Version**: *Windows*
- **Running Example**: *the playground does not have v8*

**Comment by GoodBoyDigital** [comment]
Thanks @GDX64 👍 
Will take a look into this!

**Comment by Zyie** [comment]
should be fixed in `8.0.0-alpha.3`
